tests/views_test.py

```diff
@@ -0,0 +1,57 @@
+from todo_app.data.itemStatus import ItemStatus
+from todo_app.data.todo_item import TodoItem
+from todo_app.views.view_model import ViewModel
+
+
+def test_return_done_items_only():
+    items = [
+        TodoItem('1', 'task 1', ItemStatus.TODO.value,
+                 'description of task 1'),
+        TodoItem('2', 'task 2', ItemStatus.TODO.value, ''),
+        TodoItem('3', 'task 3', ItemStatus.PROGRESS.value, ''),
+        TodoItem('4', 'task 4', ItemStatus.DONE.value, 'optional property')
+    ]
+    item_view_model = ViewModel(items)
+
+    # Get the expected list of items with 'Done' status
+    expected_done_items = [
```

Comment:
This works, but note that it basically relies on the same filtering logic as the underlying view model, which means if there was actually a subtle issue, then this is likely to miss it!

In addition, it's actually snuck into your assertions, so your test is testing itself, rather than the underlying logic!

An alternative might be to be slightly more specific, e.g.:
```
    done_items = item_view_model.done_items

    assert len(done_items) == 1
    assert done_items[0].id == "4"
    for item in done_items:
        assert item.status == ItemStatus.DONE.value
```